### PR TITLE
[grpc][openssl][upb] Fix install on uwp

### PR DESCRIPTION
**Pull request by ras0219-msft**
Fixes installation of openssl, upb, and grpc for uwp. Intends to be a stripped-down version of #10307 to quickly reenable grpc.

### CODE REVIEW

File: ports/upb/fix-uwp.patch [hunk 1]
```diff
@@ -0,0 +1,15 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 6eb8bb5..1506a64 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -13,6 +13,10 @@ cmake_policy(SET CMP0048 NEW)
+ 
+ project(upb)
+ 
++add_definitions(-D_CRT_SECURE_NO_WARNINGS)
```
Comment (JackBoosY) [inline ports/upb/fix-uwp.patch]: It looks like `_CRT_SECURE_NO_WARNINGS` only can be used in msvc. Can you please confirm that?

**Review by JackBoosY — CHANGES_REQUESTED** [review]
Please update `grpc` version info.
And please update baseline.

**Comment by JackBoosY** [comment]
I have another PR #10005 to fix upb uwp build failure.

File: ports/upb/fix-uwp.patch (hunk 1 above)
Comment (ras0219-msft) [reply ports/upb/fix-uwp.patch]: While it is necessary for MSVC, it will cause no harm for other compilers (it's just a macro that means nothing to them).

**Pull request merged**